src/main/java/core/basesyntax/db/dao/StorageDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db.dao;
+
+import core.basesyntax.model.Item;
+
+public interface StorageDao {
```

Comment:
DAO can have methods like `void addItem(String name, int quantity)` or `int getBalance(String name)` , etc.

we can not have **purchase** or **supply** words on this layer